plex_set_tracks.py

```diff
@@ -252,35 +242,30 @@ def getSeasonsFromUser(show):
                 break
             
             # Now check if they have said season in their library
-            for season in seasonNums:
-                if seasonInt == season:
-                    curSeasonIsValid = True
-                    break
+            if seasonInt in seasonNums:
+                curSeasonIsValid = True
+                break
             if curSeasonIsValid == False:
                 print("Error: Season %d of '%s' is not in your library." % (seasonInt, show.title))
                 break
                 
         # If we got through all seasons successfully, they are all valid
         if curSeasonIsValid == True:
-            allSeasonsValid = True
-    
-    # Return valid seasons to modify
-    return givenSeasonsList
+            # Return valid seasons to modify
+            return givenSeasonsList
     
 def getYesOrNoFromUser(prompt):
     """ Prompts user for a 'y' or 'n' response, then validates. 
     
         Parameters:
             prompt(str): The prompt the user will be given before receiving input.
     """
-    isValidInput = False
-    while not isValidInput:
+    while True:
```

Comment:
This one's fine, since it's clear that we're iterating until a valid result is gotten, and returning the result immediately.